models/credmark/protocols/dexes/uniswap/uniswap.py

```diff
@@ -5,6 +5,7 @@
     Contract,
     Token
```

Comment:
Should we get rid of this file to be more explicity e.g. only uniswapv2 and uniswapv3 since they're so different? I could see uniswap.py being used for the UNI token and maybe governance.